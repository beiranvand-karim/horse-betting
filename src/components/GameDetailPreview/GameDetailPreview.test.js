/* global describe, it, jest, expect*/
import {shallow} from "enzyme"
import {GameDetailPreview} from "./"
import React from "react"
import renderer from "react-test-renderer"
import {MemoryRouter} from "react-router-dom"

describe('<GameDetailPreview/>', () => {
   const gameData = {
      "@type": ".Game"
   };
   it('should render without crashing', () => {
      shallow(<GameDetailPreview gameData={gameData}/>)
   });

   it('should matches the snapshot', () => {
      const tree = renderer.create(<MemoryRouter><GameDetailPreview gameData={gameData}/></MemoryRouter>).toJSON();
      expect(tree).toMatchSnapshot()
   });
});