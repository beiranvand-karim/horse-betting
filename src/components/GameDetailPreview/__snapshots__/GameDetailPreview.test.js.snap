// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GameDetailPreview/> should matches the snapshot 1`] = `
<h4>
  game type: 
  .Game
</h4>
`;
